target/linux/ipq806x/patches-4.19/0031-mtd-add-SMEM-parser-for-QCOM-platforms.patch

```diff
@@ -18,7 +18,7 @@ Signed-off-by: Ram Chandra Jangir <rjangi@codeaurora.org>
 
```

Comment:
I can see what you've added and removed, but important is to know `Why?` you've added it and removed it. And you're missing your `Signed-off-by:` which should be the last one, after Christian's.